extensions/historical_imagery/README.md

```diff
@@ -30,9 +30,9 @@ change**
 
 ### [STAC Item Properties](https://github.com/radiantearth/stac-spec/blob/v1.0.0/item-spec/item-spec.md#properties-object)
```

Comment:
Why is this `README` file located in `historical_imagery` changed ? Is it because of some linting issue ? I don't understand why is this hyperlink `[common metadata](common-metadata.md#date-and-time-range)` changed to plain text ?

Reply:
They are just lints, yes.

Reply:
Link should have been: https://github.com/radiantearth/stac-spec/blob/v1.0.0/item-spec/common-metadata.md#date-and-time-range